Sources/Perform/Perform.swift

```diff
@@ -3,5 +3,39 @@ import Result
 import UIKit
 
 extension UIViewController {
-  public static let perform: SignalProducer<String, NoError> = .init(value: "Hello, Perform")
+  public func perform<Destination: UIViewController>(segue: Segue<Destination>, configure: (Destination) -> Void) {
```

Comment:
Ah I see. That argument is useful when you're going through the delegates, but since we're using the block, anything we pass in there, we already have access to in the block so we don't need it. I'm fine with leaving it out since I don't think there's any added value having it because of that